pkg/collector/corechecks/orchestrator/pod/pod.go

```diff
@@ -80,10 +81,12 @@ func (c *Check) Configure(
 	}
 
 	if !c.config.OrchestrationCollectionEnabled {
-		return errors.New("orchestrator check is configured but the feature is disabled")
+		log.Warn("orchestrator pod check is configured but the feature is disabled")
```

Comment:
This still produces an un-actionable warning for every installation.

Could you please use [`staticconfig`](https://github.com/datadog/datadog-agent/blob/4fe56c9b19365fe294f1683350e831a2f16b404c/pkg/autodiscovery/listeners/staticconfig.go) to conditionally enable the check if the feautre is enabled in the agent?